app/code/Magento/Checkout/i18n/en_US.csv

```diff
@@ -153,6 +153,8 @@ Shipping,Shipping
 "Maximum Number of Items to Display in Order Summary","Maximum Number of Items to Display in Order Summary"
 "Quote Lifetime (days)","Quote Lifetime (days)"
 "After Adding a Product Redirect to Shopping Cart","After Adding a Product Redirect to Shopping Cart"
+"Enable Clear Shopping Cart","Enable Clear Shopping Cart"
+"Are you sure you want to remove all items from your Shopping Cart?","Are you sure you want to remove all items from your Shopping Cart?"
```

Comment:
The phrase doesn't match the phrase in the modal. `Shopping Cart` is `shopping cart` in the modal.

Reply:
Changes made:
- Updated the phrase to `shopping cart`, translation now working.
- The `OK` phrase translation is not working in `app/code/Magento/Checkout/i18n/en_US.csv` because it is already translated in `app/code/Magento/Ui/i18n/en_US.csv` - (overwritten?).
- The `Cancel` phrase translation is not working in `app/code/Magento/Checkout/i18n/en_US.csv` because it is already translated in `app/code/Magento/Shipping/i18n/en_US.csv` - (overwritten?).